submissions/IhorOzerov/html-css-popup/index.html

```diff
@@ -0,0 +1,101 @@
+                                alt="contacts"><span>Contacts</span></a>
+                    </li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/documents.png"
+                                alt="documents"><span>Documents</span></a></li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/files.png"
+                                alt="files"><span>Files</span></a>
+                    </li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/mail.png"
+                                alt="mail"><span>Mail</span></a>
+                    </li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/maps.png"
+                                alt="maps"><span>Maps</span></a>
+                    </li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/mars.png"
+                                alt="mars"><span>Mars</span></a>
+                    </li>
+
+                    <li class="more-btn greatest"><label class="more-btn" for="more">More</label>
+                        <input id="more" type="checkbox" class="visually-hidden">
+                        <ul id="opened">
+                            <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/news.png"
+                                        alt="news"><span>News</span></a></li>
+                            <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/photos.png"
+                                        alt="photos"><span>Photos</span></a>
+                            </li>
+                            <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/promo.png"
+                                        alt="promo"><span>Promo</span></a>
+                            </li>
+                            <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/search.png"
+                                        alt="search"><span>Search</span></a>
+                            </li>
+                            <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/tanslation.png"
+                                        alt="translation"><span>Translation</span></a></li>
+                            <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/videos.png"
+                                        alt="videos"><span>Videos</span></a>
+                            </li>
+
+                        </ul>
+                    </li>
+                </ul>
+            </li>
+
+            <li> <label>
+                    <input type="checkbox" class="visually-hidden" />
+                    <img class="h-image" src="icons/bell.png" alt="bell">
+                </label>
+            </li>
+
+            <li> <label>
```

Comment:
Use Prettier to align your code

Reply:
I mean fix indentation for the whole file. What is the point to fix indentation only for one line if you have a lot of similar blocks with the same problem?)